contrib/example-backend/options/options.go

```diff
@@ -78,6 +82,7 @@ func (options *Options) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&options.KubeConfig, "kubeconfig", options.KubeConfig, "path to a kubeconfig. Only required if out-of-cluster")
 	fs.StringVar(&options.NamespacePrefix, "namespace-prefix", options.NamespacePrefix, "The prefix to use for cluster namespaces")
 	fs.StringVar(&options.PrettyName, "pretty-name", options.PrettyName, "Pretty name for the backend")
+	fs.StringVar(&options.ConsumerScope, "consumer-scope", options.ConsumerScope, "How consumers access the service provider cluster. In Kubernetes Namespaced allows namespace isolation. In kcp Cluster allows workspace isolation, and with that allows cluster-scoped resources to bind and it is generally more performant.")
```

Comment:
Suggestions:
- allow lowercase for `namespaced` and `cluster`.

CLI documentation suggestion:
> How consumers access the service provider cluster. Supported scopes are "namespaced" for namespace based isolation if service provider clusters are using Kubernetes and "cluster" for workspace based isolation if service provider clusters are using kcp.